src/main/scala/ai/privado/exporter/ExporterUtility.scala

```diff
@@ -203,11 +203,8 @@ object ExporterUtility {
     appCache: AppCache,
     ruleCache: RuleCache
   ): Option[DataFlowSubCategoryPathExcerptModel] = {
-    val allowedLineLimit = ruleCache.getSystemConfigByKey(Constants.MaxLineLimit, true)
-    val limitValue       = if (allowedLineLimit.nonEmpty) allowedLineLimit.toInt else 0
-    val sample =
-      if (limitValue != 0 && node.code.length > limitValue) node.code.take(limitValue) + "..." else node.code
-    // val sample = node.code
+    val allowedCharLimit = ruleCache.getSystemConfigByKey(Constants.MaxCharLimit, true)
+    val sample           = getTruncatedText(node.code, Some(allowedCharLimit))
```

Comment:
This should be `Option(allowedCharLimit)`

Reply:
Also a thought, can we not do `val allowedCharLimit: Option[Int] = ruleCache.getSystemConfigByKey(Constants.MaxCharLimit, true).flatMap(_.toIntOption)` ?

Reply:
As `getSystemConfigByKey` return String so instead of doing `.flatMap(_.toIntOption)`, we can directly call `toIntOption`.